WordPress/Classes/ViewRelated/NUX/WordPressAuthenticationManager.swift

```diff
@@ -314,78 +314,36 @@ extension WordPressAuthenticationManager: WordPressAuthenticatorDelegate {
     /// Presents the Login Epilogue, in the specified NavigationController.
     ///
     func presentLoginEpilogue(in navigationController: UINavigationController, for credentials: AuthenticatorCredentials, source: SignInSource?, onDismiss: @escaping () -> Void) {
```

Comment:
Rename this function and update its API doc?

Reply:
There is more than one related method in `WordPressAuthentifcatorDelegateProtocol`:

```swift
    /// Presents the Login Epilogue, in the specified NavigationController.
    ///
    /// - Parameters:
    ///   - navigationController: navigation stack for any epilogue views to be shown on.
    ///   - credentials: WPCOM or WPORG credentials.
    ///   - source: an optional identifier of the login flow, can be from the login prologue or provided by the host app.
    ///   - onDismiss: called when the auth flow is dismissed.
    func presentLoginEpilogue(in navigationController: UINavigationController, for credentials: AuthenticatorCredentials, source: SignInSource?, onDismiss: @escaping () -> Void)

    /// Presents the Login Epilogue, in the specified NavigationController.
    ///
    func presentSignupEpilogue(
        in navigationController: UINavigationController,
        for credentials: AuthenticatorCredentials,
        socialUser: SocialUser?
```

I'm thinking about finishing the work on https://github.com/wordpress-mobile/WordPress-iOS/pull/23459, which is also shown from this method,  and the Quick Start simplifications, and then coming back to this delegate (in a separate PR).

I think it should be something more generic like:

```swift
func didFinishLogin(...)
func didFiinshLogout(...)
```

I'm open to suggestions. I also want to make sure it integrates well with the new login code.

Reply:
I didn't realize it's a function of `WordPressAuthenticatorDelegate`. It's probably okay to leave it if you have to make many changes to WPAuthenticator.